lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -234,6 +234,8 @@ function dosomething_user_form_alter(&$form, $form_state, $form_id) {
     break;
 
     case 'user_profile_form':
+      $form['#validate'][] = 'dosomething_user_validate_password_length';
+      dosomething_northstar_update_user($form_state);
```

Comment:
Do you think updating the password in NS in #6231 might have been an issue with just my user account then if that is there already? @angaither